### PR TITLE
Oliver/timer overlay

### PR DESCRIPTION
**Pull request by oliharg**

### CODE REVIEW

File: samples/timer_overlay/mobile_receiver/index.html [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+    <section class="section" style="height: 1000px">
+        <div class="video-container">
+            <div class="remote-container user-container">
+                <div id="remoteVideo" class="user-video"></div>
+            </div>
+        </div>
+
+        <div class="button-container">
+            <label>Channel ID:</label>
+            <input type="atring" id="channelID" name="channelID">
```
Comment (rustyclarksonfm) [inline samples/timer_overlay/mobile_receiver/index.html]: atring -> string

File: samples/timer_overlay/index.js [hunk 2]
```diff
@@ -22,6 +23,18 @@ document.addEventListener("DOMContentLoaded", () => {
...
+
+  if (window.innerHeight < 500) {
+    height = 250;
+    width = 640;
+  }
+  else if (window.innerHeight < 1000) {
+    height = 400;
+    width = 350;
+  }
+
```
Comment (rustyclarksonfm) [inline samples/timer_overlay/index.js:37]: What's the logic here? Why are the aspect ratios varying so much? Is there some sort of portrait vs landscape hackiness going on here?

**Review by rustyclarksonfm — CHANGES_REQUESTED** [review]

File: samples/timer_overlay/index.js (hunk 2 above)
Comment (oliharg) [reply samples/timer_overlay/index.js:37]: Updated

Comment (oliharg) [reply samples/timer_overlay/index.js:37]: Yes this is to try and capture mobile vs desktop and portrait vs landscape

**Review by rustyclarksonfm — APPROVED** [review]

**Pull request merged**